src/main/java/com/minecolonies/coremod/colony/managers/BuildingManager.java

```diff
@@ -338,12 +345,36 @@ public ITownHall getTownHall()
         return townHall;
     }
 
+    @Override
+    public int getWonderMaxBuildingLevel()
+    {
+        int maxLevel = 0;
+        if(hasWonder())
+        {
+            for(final IWonder wonder : wonders)
+            {
+                int wonderLevel = wonder.getBuildingLevel();
```

Comment:
I think you can simplify this by just doing wonder.getBuildingLevel() twice, it's just as expensive as making a new var

Reply:
Updated